### DISCUSSION TITLE
Declare request example for PlainTextResponse

### DISCUSSION THREAD

**Discussion by bjhulst** [fastapi/fastapi#8726] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
PlainTextResponse
```

### Description

Could you add to this page https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/tutorial/schema-extra-example.md (or create a new page) an example of how the PlainTextResponse example value could be written in the fastapi Python code?

All it says now is "string".....  :-) 

Maybe due to a lack of a schema...but a placeholder for the actual value of the string would be usefull to know about.

### Wanted Solution

Tutorial documentation update.

### Wanted Code

```python
n/a
```

### Operating System

Other

### FastAPI Version

latest

### Python Version

3.9

**Comment by aovasylenko** [comment]
I was playing a bit with source code and in case of small changes in openapi generation utilities, and using next code:

```
from fastapi import FastAPI
from fastapi.responses import PlainTextResponse
from pydantic import BaseModel


app = FastAPI()


class HelloWorld(BaseModel):
    __root__: str

    class Config:
        schema_extra = {"example": "Hello World"}


@app.get("/", response_class=PlainTextResponse, response_model=HelloWorld)
async def main():
    return "Hello World"
```

I've received next specification:
```
openapi: 3.0.2
info:
  title: FastAPI
  version: 0.1.0
paths:
  /:
    get:
      summary: Main
      operationId: main__get
      responses:
        '200':
          description: Successful Response
          content:
            text/plain:
              schema:
                $ref: '#/components/schemas/HelloWorld'
components:
  schemas:
    HelloWorld:
      example: Hello World
      title: HelloWorld
      type: string
```

Is it what expected? (you can paste it into https://editor.swagger.io/ to check the output in UI)

**Comment by YuriiMotov** [comment]
The only way I found is to specify it in `responses`:

```python
from fastapi import FastAPI
from fastapi.responses import PlainTextResponse

app = FastAPI()


@app.get(
    "/",
    response_class=PlainTextResponse,
    responses={200: {"content": {"text/plain": {"example": "Hello, World!"}}}},
)
async def index():
    return "Hello, World!"
```

Result:
```
"responses": {
    "200": {
    "description": "Successful Response",
    "content": {
        "text/plain": {
        "schema": {
            "type": "string"
        },
        "example": "Hello, World!"
        }
    }
    }
}
```
